.env.example

```
@@ -18,6 +18,9 @@ SEND_EMAIL_HOOK_SECRETS=
 # ローカル開発: http://localhost:3000
 NEXT_PUBLIC_SITE_URL="http://localhost:3000"
 
+# ベータ版バナーの表示制御（正式リリース時にfalseに設定）
+NEXT_PUBLIC_SHOW_BETA_BANNER="true"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)